### ISSUE TITLE
Speedometer showing rubbish on Android Auto

### ISSUE THREAD

**Issue by teiksef** [osmandapp/OsmAnd#19937]
### Description

The new sppedometer is alternating between two values for all the time. Does not show correct values (or, like a broken clock, it does at some certain moments:)

### Steps to reproduce

Turning on OsmAnd in Android Auto. 

### Actual result

Even before moving, it shows 5 km/h. When moving, it alternates betwwen 5 and 8 km/h, does not show any other value.
On another ride it was alternating between 9 and 11 km/h.
No issues with navigation, directions etc - only the value shown on the speedometer is bad.

### Expected result

The speedometer should show the actual speed of the vehicle

### Your Environment (required)

OsmAnd Version: 4.7.17
Android/iOS version: Android 10
Device model: Samsung S9 plus

**Comment by EugeneZmeuk** [comment]
Today I tested the Speedometer widget for AA (Free ride) with my Samsung S22. 
It works perfectly for me. Мн device stands strictly vertically with good access to satellites. Next, I use Android API for Location source (Menu > Settings > OsmAnd Settings > Location source > Android API).

**Comment by teiksef** [comment]
I do have "Google Play services" configured as the source (I think this was the default, as I did not change it after reistalling OsmAnd a month or so ago).
I will change it to "Android API" and let you know if this works.
The navigation worked anyway, though I did have some issues a few times (like 50-100m shift in position) when run on Android Auto in parallel with Google Maps - maybe this is a likely culprit.
